numpy/core/__init__.py

```diff
@@ -53,6 +53,17 @@
 del os
 
 from . import umath
+
+# Test that multiarray is a pure python module wrapping _multiarray_umath,
+# and not the old c-extension module.
+if not (getattr(multiarray, '_multiarray_umath', None) and 
+        getattr(umath, '_multiarray_umath', None)):
```

Comment:
Any reason for not just using `hasattr` here? Are you expecting falsy values?

Reply:
fixed